tests/test_predict.py

```diff
@@ -138,9 +159,29 @@ def test_predict_with_bins_bad_values(self):
         self.assertEqual(str(raised_exceptions.exception),
                          "Empty, null, or nan are not allowed for bin values.")
 
+    def test_predict_with_bins_pil(self):
+        classifier = CustomLabelsBinningClassifier(cls_to_bin={
+            'cat': 'one',
+            'mouse': 'two',
+            'fish': 'two',
+        })
+        img1 = PIL.Image.open(EXAMPLE_CAT_IMAGE)
+        prediction_ary = classifier.predict(images=[img1])
+        self.assertEqual(len(prediction_ary), 2)
+        self.assertEqual(prediction_ary[0]['file_name'], '0')
+        names = set([pred['classification'] for pred in prediction_ary])
+        self.assertEqual(names, set(['one', 'two']))
+
+
 class TestEmbed(unittest.TestCase):
     def test_get_image_features(self):
         classifier = TreeOfLifeClassifier(device='cpu')
         self.assertEqual(classifier.model_str, 'hf-hub:imageomics/bioclip')
-        features = classifier.create_image_features_for_path(EXAMPLE_CAT_IMAGE, normalize=False)
+        features = classifier.create_image_features_for_image(EXAMPLE_CAT_IMAGE, normalize=False)
+        self.assertEqual(features.shape, torch.Size([512]))
+
+    def test_get_image_features_pil(self):
+        classifier = TreeOfLifeClassifier(device='cpu')
+        self.assertEqual(classifier.model_str, 'hf-hub:imageomics/bioclip')
+        features = classifier.create_image_features_for_image(EXAMPLE_CAT_IMAGE, normalize=False)
```

Comment:
```suggestion
        features = classifier.create_image_features_for_image(PIL.Image.open(EXAMPLE_CAT_IMAGE), normalize=False)
```
I believe this one is meant to test the output using `PIL`?